content/docs/tasks/security/mutual-tls/index.md

```diff
@@ -108,59 +76,106 @@ $ kubectl exec $(kubectl get pod -l app=httpbin -o jsonpath={.items..metadata.na
+Use the `istioctl` tool to check the effective mutual TLS settings. For example, the command below shows what mode is used for `httpbin.default.svc.cluster.local`, and what authentication policy and destination rules are used for that configuration:
 
-1. Request from `sleep` app container to `httpbin` service should succeed (return `200`)
+{{< text bash >}}
+$ istioctl authn tls-check httpbin.default.svc.cluster.local
+HOST:PORT                                  STATUS     SERVER     CLIENT     AUTHN POLICY        DESTINATION RULE
+httpbin.default.svc.cluster.local:8080     OK         mTLS       mTLS       default/            default/default
+{{< /text >}}
 
-    {{< text bash >}}
-    $ kubectl exec $(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name}) -c sleep -- curl httpbin:8000/headers -o /dev/null -s -w '%{http_code}\n'
-    200
-    {{< /text >}}
+Where:
+
+* STATUS column: shows whether the TLS settings are consistent between server (i.e `httpbin` service) and client(s), i.e all other services making call to `httpbin`.
+
+* SERVER column: shows the mode which is used on server
+
+* CLIENT column: shows the mode which is used on client(s)
+
+* AUTHN POLICY column: shows the name and namespace of the authentication policy that is used. If the policy is the mesh-wide policy, namespace is blank, e.g `default/`
+
+* DESTINATION RULE column: shows the name and namespace of the destination rule that is used.
+
+In the example output above, you can see that mutual TLS is consistently setup for `httpbin.default.svc.cluster.local` on port `8080`. The authentication policy in used is the mesh-wide policy `default`, and destination rule is `default` in `default` namespace.
 
-1. Request from `sleep` _proxy_ container to `httpbin` service on the other hand fails, as request does not use TLS nor provide a client certificate
+Now, add a service-specific destination rule for `httpbin` with incorrect TLS mode:
+
+{{< text bash >}}
+$ cat <<EOF | istioctl create -n bar -f -
+apiVersion: "networking.istio.io/v1alpha3"
+kind: "DestinationRule"
+metadata:
+  name: "bad-rule"
+spec:
+  host: "httpbin.default.svc.cluster.local"
+  trafficPolicy:
+    tls:
+      mode: DISABLE
+EOF
+{{< /text >}}
+
+Re-running the `istioctl authn tls-check` command, you will see the status is `CONFLICT`, as client is in `HTTP` mode while server is in `mTLS`.
+
+{{< text bash >}}
+$ istioctl authn tls-check httpbin.default.svc.cluster.local
+HOST:PORT                                  STATUS       SERVER     CLIENT     AUTHN POLICY        DESTINATION RULE
+httpbin.default.svc.cluster.local:8080     CONFLICT     mTLS       HTTP       default/            bad-rule/default
+{{< /text >}}
+
+As expected, requests from `sleep` to `httpbin` are now failed:
```

Comment:
I try to show what the output looks like when there are conflicts, so it's not really a step in the process. Thus, I'd like to avoid enumerate it (plus, it would require a lot of indent to align with 'previous' step). What do you think?